.github/workflows/black.yml

```diff
@@ -55,25 +52,24 @@ jobs:
           date=$(python -c "import time;print(time.localtime().tm_mday)")
           pushBranch="black-${blackVersion}-${date}"
           git push -u origin "${pushBranch}" --force
-
       - name: Make a PR
         if: steps.git-check.outputs.modified == 'true'
         env:
           access_token: ${{ secrets.GITHUB_TOKEN }}
-          file_change: ${{ steps.git-check.outputs.filechange }}
         run: |
           $blackVer = black --version
           $blackVersion = python -c "print('$($blackVer)'.split()[2])"
           $date = python -c "import time;print(time.localtime().tm_mday)"
           $pushBranch="black-$($blackVersion)-$($date)"
+          $filechange=(gc "D:/diffmod.txt").split() -join "`n`n"
           $postContents = @{title="Automated PR: Autoformat with black";
-          head="ManimCommunity:$($pushBranch)";
+          head="naveen521kk:$($pushBranch)";
           base="master";
-          body="This an automated pull requests. Files Changed are `n `n ```````n$($env:file_change)`n``````"
+          body="This an automated pull requests. Files Changed are `n `n ```````n$($filechange)`n``````"
           } | ConvertTo-Json -Compress
           echo "$postContents"
           Invoke-WebRequest `
             -Method POST `
             -Headers @{ "Accept" = "application/vnd.github.v3+json"; "Authorization"= "token $($env:access_token)"}`
-            -Uri https://api.github.com/repos/ManimCommunity/manim/pulls `
-            -Body "$($postContents)"
+            -Uri https://api.github.com/repos/naveen521kk/manim/pulls `
```

Comment:
Question: this looks like a typo, no? Must it always run on your fork?

Reply:
Ok this is a typo. Sorry.